pkg/table/tables/tables.go

```diff
@@ -532,7 +532,7 @@ func (t *TableCommon) updateRecord(sctx table.MutateContext, h kv.Handle, oldDat
 		}
 	}
 	// rebuild index
-	err = t.rebuildIndices(sctx, txn, h, touched, oldData, newData, opt.GetCreateIdxOpt())
+	err = t.rebuildIndices(sctx, txn, h, touched, oldData, newData, opt.SkipWriteUntouchedIndices, opt.GetCreateIdxOpt())
```

Comment:
How about put `SkipWriteUntouchedIndices` into `CreateIdxOpt`?

Reply:
I think It's not a good way. Because the coder can then call `Index.Create` like this:

```
index.Create(..., table.SkipWriteUntouchedIndices)
```

It is meaningless because `SkipWriteUntouchedIndices` only works for `UpdateRecord`

Reply:
If you think `rebuildIndices` has too many arguments, it maybe better to change `rebuildIndices` to `rebuildIndicesForUpdate` and pass `opt *UpdateRecordOpt` as an argument:

```go
func (t *TableCommon) rebuildIndicesForUpdate(ctx table.MutateContext, ..., opt *table.UpdateRecordOpt) error {
	...
	createIdxOpt := opt.GetCreateIdxOpt()
	for _, idx := range t.Indices() {
		...
		if untouched && opt.SkipWriteUntouchedIndices {
			continue
		}
		newVs, err := idx.FetchValues(newData, nil)
		if err != nil {
			return err
		}
		if err := t.buildIndexForRow(ctx, h, newVs, newData, asIndex(idx), txn, untouched, createIdxOpt); err != nil {
			return err
		}
	}
	return nil
}
```